user/snapper/patches/btrfs.patch

```diff
@@ -0,0 +1,17 @@
+--- a/snapper/BtrfsUtils.cc	2024-09-16 06:34:13.758280195 -0300
++++ b/snapper/BtrfsUtils.cc	2024-09-16 14:59:43.125914359 -0300
+@@ -31,15 +31,9 @@
+ #include <sys/types.h>
+ #include <sys/stat.h>
+ #include <unistd.h>
+-#ifdef HAVE_LIBBTRFS
+ #include <btrfs/ioctl.h>
+ #include <btrfs/send-utils.h>
+-#else
+-#include <linux/btrfs.h>
+-#endif
+-#ifdef HAVE_LIBBTRFSUTIL
```

Comment:
not sure what this fixes, this is already the path taken if these HAVE_ are set and they should be, if they're not then the build system has to be fixed instead..?

Reply:
i guess this is easier than making the build system define HAVE_LIBBTRFS, but the latter is defined ok afaict

Reply:
apparently these HAVE_s arent being set properly. ive tried doing a ton of stuff and it wouldnt fix this damn thing

Reply:
yeah, the former is not checked at all :)